decidim-elections/app/views/decidim/elections/trustee_zone/elections/_backup_modal.html.erb

```diff
@@ -15,10 +15,10 @@
 
   <div class="row">
     <div class="columns medium-5 medium-centered">
-      <button class="button download-election-keys">
+      <a class="button download-election-keys">
```

Comment:
Hey @agustibr ! I changed that to an `<a>` because it will be used to download things. The `href` attribute is set dynamically by the library. Related to the accessibility issues let me take a note and implement it in a future pull request 👍